try:
    a = 2/0
except Exception as erro:
    print("Aconteceu algum erro : ", erro)


'''
try:
    assadsad()
except ZeroDivisionError:
    print("Divisão por zero, não da para realizar")
except NameError:
    print("você digitou alguma coisa errada")
'''
print("Teste")
